src/recipe/admin/src/module.ts

```diff
@@ -135,25 +168,13 @@ export default defineNuxtModule<ModuleOptions>({
         }
       }),
       defaultLocale: app?.i18n?.defaultLanguage?.code,
-      detectBrowserLanguage: {
-        useCookie: true,
-        cookieKey: 'i18n_cookie'
-      }
-    });
-    await installModule("@nuxtjs/tailwindcss", {
-      exposeConfig: true,
-      cssPath: resolver.resolve("./runtime/assets/tailwind.css"),
+    };
+    _nuxt.options.tailwindcss ||= {
       config: {
-        content: {
-          files: [
-            resolver.resolve("./runtime/components/**/*.{vue,mjs,ts}"),
-            resolver.resolve("./runtime/*.{mjs,js,ts}")
-          ]
-        },
         theme: {
           screens: _options.composables.useBreakpoints.screens
         }
       }
-    })
+    };
```

Comment:
this is probably not working as expected. during tests either `_nuxt.options.tailwindcss` was null (which shouldn't be) or righthand side is not set so breakpoints are not changing.